front/lib/api/assistant/global_agents.ts

```diff
@@ -123,6 +123,20 @@ async function getDataSourcesAndWorkspaceIdForGlobalAgents(
   };
 }
 
+function _getDefaultSearchActionForGlobalAgent({
+  agentSid,
```

Comment:
@overmode belated and small nit: we use `[ObjectName]Id` in general in the rest of the codebase. Here it would be `agentConfigurationId` or `agentId` 

Mind quickly modifying it? :pray: